src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java

```diff
@@ -70,7 +70,8 @@
 
     private final KafkaRouteExFW routeExRO = new KafkaRouteExFW();
     private final KafkaBeginExFW beginExRO = new KafkaBeginExFW();
-    private final OctetsFW octetsRO = new OctetsFW();
+    private final OctetsFW octetsRO1 = new OctetsFW();
+    private final OctetsFW octetsRO2 = new OctetsFW();
```

Comment:
Suggest renaming to `valueRO` and `keyRO`.

Reply:
done